molecule/default/playbook.yml

```diff
@@ -9,4 +9,6 @@
       listen: ssl certificate changed
       # Use command so that we get an error if the handler is triggered during
       # the idempotence check
```

Comment:
Can you update this comment to match the change below

Reply:
Is the behavior the same? Using `command` effectively led the Molecule 2 tests to complain with

```
    RUNNING HANDLER [Create file when ssl certificate changed] *********************
 [WARNING]: Consider using the file module with state=touch rather than running
touch.  If you need to use command because file is insufficient you can add
warn=False to this command task or set command_warnings=False in ansible.cfg to
get rid of this message.
````

but not fail so I could also revert to the previous form.

Reply:
I think so

Reply:
:+1: updated in 0b5b96c